Tests/MongoSwiftTests/ChangeStreamTests.swift

```diff
@@ -3,8 +3,98 @@ import mongoc
 import Nimble
 import XCTest
 
-final class ChangeStreamTest: MongoSwiftTestCase {
-    func testChangeStream() throws {
+final class ChangeStreamTests: MongoSwiftTestCase {
+    func testChangeStreamOnAClient() throws {
+        guard MongoSwiftTestCase.topologyType != .single else {
+            print("Skipping test case because of unsupported topology type \(MongoSwiftTestCase.topologyType)")
+            return
+        }
+
+        let client = try MongoClient()
+        guard try client.serverVersion() >= ServerVersion(major: 4, minor: 0) else {
+            print("Skipping test case for server version \(try client.serverVersion())")
+            return
+        }
+
+        let changeStream = try client.watch()
+        let db1 = client.db("db1")
+        defer { try? db1.drop() }
+        let coll1 = db1.collection("coll1")
+        let coll2 = db1.collection("coll2")
+        try coll1.insertOne(["a": 1])
+        try coll2.insertOne(["x": 123])
+
+        let res1 = changeStream.next()
+        // expect the change stream to contain a change document for the `insert` operation
+        expect(res1).toNot(beNil())
```

Comment:
there seem to be a lot of cases in these tests where you check the same few fields of a change stream document - consider introducing some kind of helper or helpers where you pass in a change stream document and expected values for a few fields and it does the comparison